packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -251,13 +254,18 @@ const baseConfig = (target) => {
                                 loader: findDepInStack('source-map-loader')
                             }
                         },
-                        {
-                            test: /universal[\\/]+extensibility[\\/]+extensions/,
-                            loader: `@salesforce/pwa-kit-dev/configs/webpack/loaders/extensions-loader`,
-                            options: {
-                                pkg
+                        ruleForApplicationExtensibility({
+                            loaderOptions: {
+                                appConfig: getConfig(),
+                                target: 'web'
+                            }
+                        }),
+                        ruleForApplicationExtensibility({
```

Comment:
nit, there should never have a webpack build for both `node` and `browser`

```suggestion
                        target === 'web' && ruleForApplicationExtensibility({
                            loaderOptions: {
                                appConfig: getConfig(),
                                target: 'web'
                            }
                        }),
                        target === 'node' && ruleForApplicationExtensibility({
```

Reply:
I'll create a ticket for this. The solution here is going to be just moving these from the base in to the client and server